components/PublishPage/PublishHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { useContext } from 'react'
+import Link from 'next/link'
+import { ChevronRightIcon } from '@heroicons/react/20/solid'
+
+import LoggedInButtonPopUp from '@/components/LoggedInButtonPopUp'
+import Logo from '@/components/Logo'
+
+import { PublishContext } from '@/context/PublishContext'
+
+const steps = ['Compile', 'Mint', 'Complete']
```

Comment:
To me "Details" makes more sense imo.

```suggestion
const steps = ['Details', 'Mint', 'Complete']
```